src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -57,136 +40,172 @@ private ExpressionSyntax ParseInterpolatedStringToken()
             var originalText = originalToken.ValueText; // this is actually the source text
             Debug.Assert(originalText[0] == '$' || originalText[0] == '@');
 
-            var isAltInterpolatedVerbatim = originalText.Length > 2 && originalText[0] == '@'; // @$
```

Comment:
diff is awful. i'm not sure a good way to review before/after. i recommend just trying to understand the after part.